nuxt.config.ts

```diff
@@ -1,10 +1,12 @@
+import { defineNuxtConfig } from "nuxt/config";
 import fetchEvents from "./hooks/update-events";
 import fetchAdvocates from "./hooks/update-advocates";
 import fetchEcosystemMembers from "./hooks/update-ecosystem";
 import pkg from "./package.json";
 
 const { AIRTABLE_API_KEY, GENERATE_CONTENT, NODE_ENV, SITE_URL } = process.env;
 const IS_PRODUCTION = NODE_ENV === "production";
+const siteUrl = SITE_URL || "https://qiskit.org";
```

Comment:
Do we need this to be variable?

I'm thinking that maybe we do because of our testing environment, but I'm not sure if it makes a difference or if we could just set it to "https://qiskit.org" always.

Reply:
There was [an issue](https://github.com/Qiskit/qiskit.org/issues/3235) for change this from static to variable. Here, I just reorder it